### PR TITLE
fix VersionInfoSpec, UpdateFileForUpgrade e2e tests in NetworkAdminService

### LINKED ISSUE #7680: Run NetworkAdmin e2e tests successfully
Freeze Service e2e tests:
hedera-services/hedera-node/test-clients/src/main/java/com.hedera.services/bdd/suites/freeze
- [ ] CryptoTransferThenFreezeTest
- [ ] FreezeAbort
- [ ] FreezeDockerNetwork
- [ ] FreezeIntellijNetwork
- [ ] FreezeSuite
- [ ] FreezeUpgrade
- [ ] PrepareUpgrade
- [ ] SimpleFreezeOnly
- [ ] UpdateFileForUpgrade
- [ ] UpdateServerFiles
- [ ] UpgradeSuite

Other NetworkAdmin Service e2e tests:
- [x] meta/VersionInfoSpec
- [ ] utilops/checks/VerifyGetFastRecordNotSupported (part of UnsupportedQueriesRegression.java)
- [ ] NetworkGetAccountDetails (part of RandomOps.java)
- [ ] maybe some others, need to look around for them

Need to create e2e tests for these handlers:
- [ ] NetworkGetByKey
- [ ] NetworkGetExecutionTime
- [ ] NetworkTransactionGetReceipt
- [ ] NetworkTransactionGetRecord
- [ ] NetworkUncheckedSubmit

### PR DESCRIPTION
**Pull request by kimbor**
**Description**:
Fix some networkAdminService e2e tests:

- Enable VersionInfoSpec
- Enable and fix UpdateFileForUpgrade 

**Related issue(s)**:

Partially fixes #7680

### CODE REVIEW

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/utils/FileServiceUtils.java [hunk 1]
```diff
@@ -81,16 +80,20 @@ public static void validateContent(@NonNull byte[] content, @NonNull FilesConfig
             throw new PreCheckException(INVALID_FILE_ID);
         }
 
-        final var fileMeta = fileStore.getFileMetadata(fileId);
-        mustExist(fileMeta, INVALID_FILE_ID);
+        final var fileConfig = context.configuration().getConfigData(FilesConfig.class);
+
+        // TODO we should still check if upgrade file exist after modularization is done otherwise E2E test is failing.
```
Comment (mhess-swl) [inline hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/utils/FileServiceUtils.java:85]: `@future('####')` instead?

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/freeze/CommonUpgradeResources.java [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 public final class CommonUpgradeResources {
     public static final String DEFAULT_UPGRADE_FILE_ID = "0.0.150";
-    public static final String DEFAULT_UPGRADE_FILE_PATH = "upgrade.zip";
+    public static final String DEFAULT_UPGRADE_FILE_PATH = "testfiles/poeticUpgrade.zip";
```
Comment (iwsimon) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/freeze/CommonUpgradeResources.java:32]: Why is the filename changed here? Does it has any impact on other places?

Comment (kimbor) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/freeze/CommonUpgradeResources.java:32]: The old upgrade.zip file was deleted quite awhile ago during a reorganization of the test directory structure. I don't know whether it was deleted accidentally or on purpose, but no file with that name currently exists anywhere in the project. All I'm doing here is changing the test to use a different file which does still exist.

**Review by povolev15 — DISMISSED** [review]
LGTM

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/freeze/CommonUpgradeResources.java (hunk 2 above)
Comment (iwsimon) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/freeze/CommonUpgradeResources.java:32]: This value is also used in PerfTestLoadSettings. I am not familiar with what this setting is used for.

File: hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/utils/FileServiceUtils.java (hunk 1 above)
Comment (kimbor) [reply hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/utils/FileServiceUtils.java]: Fixed

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/freeze/CommonUpgradeResources.java (hunk 2 above)
Comment (kimbor) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/freeze/CommonUpgradeResources.java:32]: In PerfTestLoadSettings, DEFAULT_UPGRADE_FILE_PATH is never read except to output it in a toString() method. The way it was before (pointing to a non-existent file upgrade.zip) would have been broken. I think it's safe to change it here. It was definitely broken before.

**Review by mhess-swl — APPROVED** [review]

**Pull request merged**